packages/react/src/components/Table/Table.test.jsx

```diff
@@ -565,81 +563,6 @@ describe('Table', () => {
     expect(wrapper.find('.bx--search-input').prop('value')).toEqual('');
   });
 
-  it('cells should always wrap by default', () => {
```

Comment:
It scares me when I see people refactoring code AND removing unit tests.  The unit tests are there to ensure that you preserve the same behaviour before and after the refactor.  Now we don't know and I have to believe that your new tests effectively capture the same behaviour.  I would recommend leaving in the existing unit tests until such time that we are confident that your new code and old code behave the same.

Reply:
We had this discussion for these tests almost a year ago :-) 
See https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1645#discussion_r499643701
This is for v3, just to be extra safe.

Reply:
Ok, but this never did address my original concerns :(  ie, you concluded that the risk is acceptable to you.

My question still remains... did the unit tests HAVE to be removed at this point, or could they have remained and be removed AFTER the refactor has be validated.  The one thing unit tests are really got at, is catching regressions during a refactor operation.  I'm all for refactoring code, I do it all the time.  But if I need to rewrite tests as a result of the refactor then my confidence drops, since, I can't say that my changes, under the hood, still have the same behaviours before and after my changes.

Reply:
Short answer yes. 
Because these unit tests were based on checking props (not functionality) of _internal sub components_, e.g.
```
// Make sure subcomponent TableBodyRow has prop wrapCellText='always'
`expect(wrapper2.find(TableBodyRow).prop('options').wrapCellText).toEqual('always');`
```
Not a great test to begin with, I know. Now, since this refactoring removes that wrapCellText prop, it can't be used for testing anymore. The actual functionality is covered by other tests added in the previous PR.